File: lib/web_utils.php
<?php


function http_post_param($name)
{
	if(!array_key_exists($name,$_POST)) return '';
	return htmlspecialchars($_POST[$name]);
}


function http_get_param($name)
{
	if(!array_key_exists($name,$_GET)) return '';
	return htmlspecialchars($_GET[$name]);
}




?>

File: lib/web_utils.py
import cgi

def http_post_param(name):
	return cgi.escape(app.form.getfirst(name,''),True)

def http_get_param(name):
	if name not in app.get_params:
		return ''
	return cgi.escape(app.get_params[name],True)


